src/main/java/org/jenkinsci/plugins/workflow/support/storage/SimpleXStreamFlowNodeStorage.java

```diff
@@ -138,6 +141,14 @@ private Tag(@Nonnull FlowNode node, @Nonnull List<Action> actions) {
             this.actions = actions.isEmpty() ? null : actions.toArray(new Action[actions.size()]);
         }
 
+        private void storeActions() {  // We've already loaded the actions, may as well store them to the FlowNode
+            try {
+                FlowNode_setActions.invoke(this.node, actions());
+            } catch (InvocationTargetException|IllegalAccessException ex) {
+                throw new Error(ex);
```

Comment:
@rsandell Fair enough (I think it probably reflects something at that level if the class changes in such a way), but can just convert to a RunTimeException.